frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -518,6 +525,14 @@ export default function SurveyEdit(): JSX.Element {
                                                                         </Field>
                                                                     </div>
                                                                 )}
+                                                                <Field name="buttonText" label="Button text">
+                                                                    <LemonInput
+                                                                        value={
+                                                                            question.buttonText ||
```

Comment:
oh nvm found it!

The `||` here means whenever I empty the buttonText it is immediately replaced by the text in `submitButtonText` - probably annoying for existing users, is this for backwards compatibility?

Reply:
yes for backwards compatibility, let me see if I can fix that though